CONTRIBUTING.md

```diff
@@ -118,11 +118,10 @@ python manage.py runserver
 
 # Run unit tests
 
-Instructions on how to run all test code:
+Execute unit tests by running the following command from within the `project` directory.
 
 ```py
-cd project && python manage.py test
-```
+python manage.py test
```

Comment:
We still want the code fences around the command.


> \`\`\`sh
>
>python manage.py test
>\`\`\`

Reply:
They somehow got removed from the code suggestion previously.

Reply:
Oh okay. Used py because everything above it is in py. So i should use sh right? Thanks for your help

Reply:
Yes. The command is technically run in a terminal, which is a shell environment rather than Python. I.e., we are calling the `python` command and passing in some arguments from within the operating system terminal/shell.